pkgs/development/libraries/tachyon/default.nix

```diff
@@ -36,8 +36,8 @@ stdenv.mkDerivation rec {
   '';
   arch = if stdenv.hostPlatform.system == "x86_64-linux"   then "linux-64-thr"  else
          if stdenv.hostPlatform.system == "i686-linux"     then "linux-thr"     else
-         if stdenv.hostPlatform.system == "aarch64-linux"  then "linux-arm-thr" else
-         if stdenv.hostPlatform.system == "armv7l-linux"   then "linux-arm-thr" else
+         if stdenv.hostPlatform.system == "aarch64-linux"  then "linux-arm"     else
```

Comment:
Please add a comment that explains why threading is disabled. You could just link to this thread. That might be helpful when somebody wants to work on the derivation in the future.

Reply:
Good idea, I've added a comment. Thanks!